cognee/tasks/ingestion/transform_data.py

```
@@ -8,11 +8,11 @@ def get_data_from_llama_index(data_point: Union[Document, ImageDocument], datase
     if type(data_point) == Document:
```

Ruff (0.8.0)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)